src/tracing/index.ts

```diff
@@ -16,6 +17,14 @@ window.GIT_COMMIT_HASH = process.env.REACT_APP_GIT_COMMIT_HASH
 const AMPLITUDE_DUMMY_KEY = '00000000000000000000000000000000'
 export const STATSIG_DUMMY_KEY = 'client-0000000000000000000000000000000000000000000'
 
+const SENTRY_USER_ID_KEY = 'sentry-user-id'
+
+let sentryUserId = localStorage.getItem(SENTRY_USER_ID_KEY)
+if (!sentryUserId) {
+  sentryUserId = v4uuid()
+  localStorage.setItem(SENTRY_USER_ID_KEY, sentryUserId)
+}
+
```

Comment:
This is unused before Sentry.init, so rather than split it from its usage, move it below the Sentry.init to be colocated with Sentry.setUser.